src/containers/ConversionForm/index.tsx

```diff
@@ -80,18 +91,71 @@ const ConversionForm = ({
                         setFileToConvert(file);
                     }}
                     customRequest={(options) =>
-                        customRequest(fileToConvert, receiveFileToConvert, setError, options)
+                        customRequest(
+                            fileToConvert,
+                            receiveFileToConvert,
+                            setError,
+                            options
+                        )
                     }
                 >
                     <Button type="default">Select file</Button>
                 </Upload>
             </div>
-            <Divider orientation="right" orientationMargin={400}>
-                {" "}
-            </Divider>
+            <Collapse accordion className={styles.collapse} bordered={false}>
+                <Panel
+                    key="placeholder"
+                    header=""
+                    showArrow={false}
+                    collapsible="disabled"
+                ></Panel>
+                <Panel
+                    key="placeholder"
+                    header=""
+                    showArrow={false}
+                    collapsible="disabled"
+                ></Panel>
+            </Collapse>
+            {fileToConvert ? (
+                <div>
+                    <h3 className={styles.provide}>
+                        Provide display information (optional)
+                    </h3>
+                    <h3 className={styles.convertText}>
+                        You can import your model now with defaults, or specify
+                        how you want your Smoldyn trajectory displayed below.
+                    </h3>
+                    <h3 className={styles.provide}> </h3>
+                    <h3 className={styles.convertText}>Trajectory title</h3>
+                    <Input placeholder="Start typing..." />
+                    <Collapse
+                        accordion
+                        className={styles.collapse}
+                        bordered={false}
+                    >
+                        <Panel
+                            key="placeholder"
+                            header=""
+                            showArrow={false}
+                            collapsible="disabled"
+                        ></Panel>
+                        <Panel
+                            key="placeholder"
+                            header=""
+                            showArrow={false}
+                            collapsible="disabled"
+                        ></Panel>
+                    </Collapse>
+                </div>
+            ) : null}
+            {conversionProcessingData.templateMap ? (
+                <InputForm
+                    conversionProcessingData={conversionProcessingData}
```

Comment:
you already have conditional rendering in this form. And there are multiple conditional renders in here which makes the logic harder to follow. I'm going to move all the things that only should be rendering after you upload a file into the form component